src/main/java/app/service/FileWriterService.java

```diff
@@ -0,0 +1,8 @@
+package app.service;
+
+import app.model.SupplyFruit;
+import java.util.List;
+
+public interface FileWriterService {
+    boolean writeData(List<SupplyFruit> fruits, String filePath);
```

Comment:
SupplyFruit objects also hold dates. You do not need to operate with them in the result file.